modules/edu.gemini.epics.acm/src/main/java/edu/gemini/epics/acm/CaService.java

```diff
@@ -5,6 +5,7 @@
 
```

Comment:
Just out of my own personal random curiosity, why are we still using Java code here?